pywbemtools/pywbemcli/_cmd_statistics.py

```diff
@@ -0,0 +1,347 @@
+    ``WBEMServerResponseTime``.
+    """
+    # pylint: disable=too-many-function-args
+    context.execute_cmd(lambda: cmd_statistics_reset(context))
+
+
+@statistics_group.command('server-on', cls=PywbemcliCommand,
+                          options_metavar=CMD_OPTS_TXT)
+@add_options(help_option)
+@click.pass_obj
+def statistics_server_on(context):
+    """
+    Enable statistics on current server.
+
+    This may fail if the server does not manage statistics or does not allow
+    a client to modify the state of statistics gathering.
+
+    This does not impact pywbem local client statistics gathering. See the
+    --timestats general option.
+    """
+    # pylint: disable=too-many-function-args
+    context.execute_cmd(lambda: cmd_statistics_server_on(context))
+
+
+@statistics_group.command('server-off', cls=PywbemcliCommand,
+                          options_metavar=CMD_OPTS_TXT)
+@add_options(help_option)
+@click.pass_obj
+def statistics_server_off(context):
+    """
+    Disable statistics on current server.
+
+    This may fail if the server does not manage statistics or does not allow
+    a client to modify the state of statistics gathering.
+
+    This does not impact pywbem local client statistics gathering. See the
+    --timestats general option.
+    """
+    # pylint: disable=too-many-function-args
+
+    context.execute_cmd(lambda: cmd_statistics_server_off(context))
+
+
+@statistics_group.command('server-show', cls=PywbemcliCommand,
+                          options_metavar=CMD_OPTS_TXT)
+@add_options(help_option)
+@click.pass_obj
+def statistics_server_show(context):
+    """
+    Display statistics gathered by server.
+
+    Display the current statistics gathered by the server if statistics
+    gathering is implemented for the current WBEM server.
+
+    Presents a table of the gathered statistical information.
+
+    This is not implemented. See issue #895
+    """
+    # pylint: disable=too-many-function-args
+    context.execute_cmd(lambda: cmd_statistics_server_show(context))
+
+
+@statistics_group.command('show', cls=PywbemcliCommand,
+                          options_metavar=CMD_OPTS_TXT)
+@add_options(help_option)
+@click.pass_obj
+def statistics_show(context):
+    """
+    Display statistics gathered by client.
+
+    Display the current statistics including local client gathered statistics
+    on WBEM operations and if server statistics are enabled, the
+    WBEMServerResponseTime statistics.
+
+    Statististics are always displayed as a table independent of output-format
+    defined.
+    """
+    # pylint: disable=too-many-function-args
+    context.execute_cmd(lambda: cmd_statistics_show(context))
+
+
+@statistics_group.command('status', cls=PywbemcliCommand,
+                          options_metavar=CMD_OPTS_TXT)
+@add_options(help_option)
+@click.pass_obj
+def statistics_status(context):
+    """
+    Show statistics enabled status for client and server.
```

Comment:
Make the short description consistent with the short description shown in the command summary starting at line 1600 of `commands.rst`.